docs/blocked-export.md

```diff
@@ -1,12 +1,25 @@
 These are export PRs which the exporter should have merged by now.
 
-Usually this is caused by failing Travis checks. Read the Travis logs to understand the failure.
+Usually this is caused by failing TaskCluster checks. Read the TaskCluster logs to understand the failure.
 
-A common cause of failure is that [when many tests are affected, Travis jobs will time out](https://github.com/web-platform-tests/wpt/issues/7660). The timeout is at around 50 minutes. When this happens, the PR has to be force merged. Link to [#7660](https://github.com/web-platform-tests/wpt/issues/7660) in a comment and ask @stephenmcgruer or @Hexcles to merge the PR.
+A common problem is the TaskCluster stability check failing, which indicates
+that a test is flaky. This is very often a problem with the test itself and will
+require help from the author, so unflaking these can take some time. To avoid
+these PRs from piling up we aim to clear them from the queue and apply a fix in
+the background.
+
+Follow these steps (see [#23617](https://github.com/web-platform-tests/wpt/pull/23617) for a recent example):
+- copy the "Unstable Results" from the TaskCluster log into the PR for easier
+  viewing.
+- create a crbug to record the flake, which should point to the PR and be assigned to the CL author.
+- reference the new bug in the PR.
+- force merge the PR.
+
+Another common cause of failure is that [when many tests are affected, TaskCluster jobs will time out](https://github.com/web-platform-tests/wpt/issues/7660). The timeout is at around 50 minutes. When this happens, the PR has to be force merged. Link to [#7660](https://github.com/web-platform-tests/wpt/issues/7660) in a comment and ask @foolip or @Hexcles to merge the PR.
```

Comment:
Did you intend to change the PoC? @stephenmcgruer is also an admin.

Reply:
Unintended (bad rebase) -- good catch!